app/controllers/admin/words_controller.rb

```diff
@@ -0,0 +1,84 @@
+class Admin::WordsController < ApplicationController
+  layout 'layouts/admin'
+  before_action :get_word, only:[:edit, :update, :destroy]
+
+  def index
+    @words = Word.paginate(page: params[:page], per_page: Settings.paginate)
+  end
+
+  def new
+    @word = Word.new
+    @courses = Course.all
+    @lessons = []
+    if params[:course_id].present?
+      @lessons = Course.find(params[:course_id]).lessons
+    end
+    if request.xhr?
+      respond_to do |format|
+        format.json {
+          render json: {lessons: @lessons}
+        }
+      end
+    end
+  end
+
+  def create
+    @word = Word.new(word_params)
+    if @word.save
+      flash[:success] = "Add success"
+      redirect_to admin_words_path
+    else
+      @lessons = Lesson.all
+      flash[:danger] = "Add fail"
+      render :new
+    end
+  end
+
+  def edit
+    @courses = Course.all
+    @lessons = []
+    if params[:course_id].present?
+      @lessons = Course.find(params[:course_id]).lessons
+    end
+    if request.xhr?
```

Comment:
```suggestion
    @lessons = Lesson.where(course_id: params ....)
    if request.xhr?
```
biến @lessons và lessons ở concept này khác gì nhau

Reply:
trả lời đi chứ em

Reply:
@lessons sử dụng khi muốn truyền biến đó ra view còn lessons chỉ dùng cục bộ trong controller

Reply:
vậy ở đây, biến nào dùng cho js biến nào dùng cho html